### ISSUE TITLE
Nested Namespace support.

### ISSUE THREAD

**Issue by michaelcheers** [bridgedotnet/Bridge#1430]
### Expected

`Hello World` string should be printed in Console.

### Actual

```
Error:
C# to JavaScript Compilation Failed
Bridge.Contract.EmitterException: Nested namespaces are not supported
...
```

### Steps To Reproduce

[Deck](http://deck.net/0c687074aaea86e05454a8542179284c)
[.NET Fiddle](https://dotnetfiddle.net/GD4b7V)

```cs
namespace OuterNamespace
{
    using System;
    
    public class Program
    {
        public static void Main()
        {
            var msg = InnerNamespace.Constants.TestConst;
            Console.WriteLine(msg);
        }
    }
    
    namespace InnerNamespace
    {
        public class Constants
        {
            public const string TestConst = "Hello World!";
        }
    }
}
```

**Comment by LeonidVeriga** [comment]
@michaelcheers, can you please elaborate this feature request, it would be great to have a C# sample and expected JavaScript output.

**Comment by AndreyChechel** [comment]
A sample has been added.

**Comment by geoffreymcgill** [comment]
This feature has been added and will be included in the upcoming Bridge 15.6 release.
